lib/ecto/adapters/postgres/connection.ex

```diff
@@ -289,6 +294,8 @@ if Code.ensure_loaded?(Postgrex) do
               {_, nil}, acc ->
                 acc
 
+              {:format, value}, acc ->
+                [String.upcase("#{format_to_sql(value)}") | acc]
```

Comment:
We don't format the repo yet but here is what the formatter would do:

```suggestion
                [String.upcase("#{format_to_sql(value)}") | acc]

```

:)